shared/steps/assess/check_if_file_in_repo_exists.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package assess
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"code.gitea.io/sdk/gitea"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/open-component-model/ocm-e2e-framework/shared"
+)
+
+// CheckFileInRepoExists adds a check to verify that file(s) exists in a repository
+func CheckFileInRepoExists(files ...File) features.Func {
```

Comment:
This already exists when checking for empty content. Why would you need to check for a file but not the content in a test environment?

Reply:
I would also make sure that the content still matches / was updated during the merge.

Reply:
We are testing the results of that anyway in the following steps? Product deployment and values file depend on the config and we will add more tests for deployment to another cluster, deployment to another namespace, multiple deployments, etc. in the next tickets. It will end up being super tightly coupled

Reply:
Okay then.